src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs

```diff
@@ -101,6 +107,15 @@ public static class AddPackageReferenceCommand
             });
         }
 
+        private static void ValidatePrerelease(bool prerelease, bool noVersion, string commandName)
```

Comment:
If it's only 1 time used what is the merit of having separate method with 1 logic?

Reply:
We use similar functions to validate in the same file, I think its easier to read this way